modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/TypeScriptClientCodegen.java

```diff
@@ -36,7 +36,7 @@
 import static org.openapitools.codegen.utils.StringUtils.underscore;
 
 
-public class TypeScriptClientCodegen extends DefaultCodegen implements CodegenConfig {
+public class TypeScriptClientCodegen extends AbstractTypeScriptClientCodegen implements CodegenConfig {
```

Comment:
Hmm. I believe the goal of this TypescriptClientCodegen project was to have a greenfield implementation, not burden by backward compatibility with existing TS generators. Ultimately making it easier to make changes in it.

@TiFu any thoughts?